README.md

```diff
@@ -1,22 +1,33 @@
 # ChugSplash
 
-A declarative and deterministic framework for deploying and upgrading smart contracts. Available as a Foundry and Hardhat plugin.
+A declarative and deterministic framework for deploying and upgrading smart contracts. Available for both Hardhat and Foundry.
+
+Powers [ChugSplash Managed](https://www.chugsplash.io).
 
 > **WARNING**: The code and contracts ChugSplash uses to deploy and upgrade your contracts HAVE NOT been audited. ChugSplash is a BETA product undergoing significant active development. ChugSplash's behavior and APIs are subject to change at any time at our discretion. You should not use ChugSplash if you would be very upset with your project breaking without notice. We make no guarantees about the safety of any contract deployments using the ChugSplash system.
 
 If you want to use ChugSplash in production, ask a question, or request a feature then we'd love to hear from you in the [Discord!](https://discord.com/invite/CqUPhgRrxq)
 
-## Key Features
+## Features
 
-- Deploy new upgradeable contracts
-- Upgrade existing contracts
-- Fully deterministic and idempotent deployments
-- Declarative syntax for a better developer experience
-- Compatible with contracts deployed using the [OpenZeppelin Hardhat Upgrades API](https://docs.openzeppelin.com/upgrades-plugins/1.x/api-hardhat-upgrades)
+### ChugSplash Core Protocol
```

Comment:
I think we should have a blurb here that gives a bit of context. Something like: "The Core Protocol handles the entire deployment process. It includes smart contracts as well as a Hardhat and Foundry plugin. All of the code is inside this repository."